reinforcement_learning/common/start_training.py

```diff
@@ -0,0 +1,37 @@
+# BEGIN--------------------PROJECT-ROOT-PATH-APPENDING-------------------------#
+import sys
+import os
+
+REL_PROJECT_ROOT_PATH = "./../../"
+ABS_FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+ABS_PROJECT_ROOT_PATH = os.path.normpath(os.path.join(ABS_FILE_DIR, REL_PROJECT_ROOT_PATH))
+sys.path.append(ABS_PROJECT_ROOT_PATH)
+# -------------------------PROJECT-ROOT-PATH-APPENDING----------------------END#
+
+from environments.tic_tac_toe.tic_tac_toe_engine import TicTacToeEngine
+from reinforcement_learning.base.base_agent import BaseAgent
+from reinforcement_learning.agents.basic_mc_agent.basic_mc_agent import BasicAgent
+from reinforcement_learning.agents.q_learning_agent.q_learning_agent import QLearningAgent
+from reinforcement_learning.common.simple_training import SimpleTraining
+
+if __name__ == '__main__':
+    # To start a training you need an engine:
+    engine = TicTacToeEngine(2, 3, 3)
+
+    # ...and agents:
+    agents = [BasicAgent(), QLearningAgent(0.3, 0.1, 0.9)]
```

Comment:
Maybe we should add also optional creating of the agents here?

Reply:
What do you mean?

Reply:
Instead of create_agents.py

Reply:
okey, I'll do

Reply:
Hm, after short pondering I thing that there is saving already (which is same as creating), so let's stay with this version.

Reply:
ok